app/scoring/page.tsx

```diff
@@ -8,6 +8,24 @@ interface Dictionary<T> {
     [Key: string]: T;
 }
 
+
+function generateContestantRoundScores(contestantTeamsList: Team[], numberOfRounds: number) {
+
+    const contestantRoundScores: number[] = []
+
+    for(let i = 0; i < numberOfRounds; i++) {
+        const roundScore = contestantTeamsList.reduce(
+            (acc: number, x: Team) => {
+                const teamShouldBeScored = shouldBeScored(contestantTeamsList, x, i)
```

Comment:
For some reason Github suggestions aren't available here 😅 but I'm thinking something like `team` or `contestantTeam` instead of `x` here.

Reply:
Ah, it's already merged 😅